task-manager/src/test/java/TaskTest.java

```diff
@@ -0,0 +1,84 @@
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.*;
+import org.taskmanager.Task;
+import org.taskmanager.TaskPriority;
+
+import java.time.LocalDate;
+
+public class TaskTest {
+
+    private Task task;
+
+    @BeforeEach
+    public void setUp() {
+        task = new Task("Title", "Description", LocalDate.now(), TaskPriority.HIGH);
+    }
+
+    @Test
+    public void testTaskCreation() {
+        assertNotNull(task);
+

```

Comment:
Is this Not Null check really necessary? I can see that you are testing some things later that make sure the task is not null.